curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660ef19b95d3308e7dd31bb6.md

```diff
@@ -20,6 +20,12 @@ Print the value of the `character` variable to the console. Then, click the "Con
 
 # --hints--
 
+You should declare a variable `character` and assign it a value.
+
+```js
+assert.match(__helpers.removeJSComments(code), /let\s+character\s*=\s*[^;]+;/);
```

Comment:
please check if the variable exists and that it has the right value, do not use regex when there are other methods